packages/desktop/components/drawers/network-config/drawers/ConfirmLedgerEvmAddressDrawer.svelte

```diff
@@ -10,7 +10,7 @@
 
     export let drawerRouter: Router<DrawerRoute>
 
-    $: address = $selectedAccount?.evmAddress
+    $: address = $selectedAccount?.evmAddresses[60]
```

Comment:
can you do the following to remove the magic number?
```suggestion
    $: address = $selectedAccount?.evmAddresses[$selectedChain.cointype]
```